src/ledger/utils.ts

```diff
@@ -101,38 +100,57 @@ function compareTransactions(
   return first.outcome.ledgerVersion < second.outcome.ledgerVersion ? -1 : 1
 }
 
-function hasCompleteLedgerRange(
+async function hasLedgerVersions(connection: Connection, min: number, max: number | undefined): Promise<boolean> {
+  const response = await connection.request({
+    command: 'server_info'
+  })
+  const ledgerRange = response.result.info.complete_ledgers
+  const rangeset = new RangeSet()
+  rangeset.parseAndAddRanges(ledgerRange)
+  if (!max) {
+    return rangeset.containsValue(min)
```

Comment:
This condition will trigger when max=0, and I'm not sure if that's intentional. If it is could we break this into a more explicit condition such as `max == undefined || max == 0`

Reply:
I'm not really sure because I don't understand the getter/setter paradigm, but messing with this is out of the scope of this PR.

Reply:
Turns out the method isn't used in the repo, so I just deleted it.